app/presenters/order_detail_presenter.rb

```diff
@@ -49,6 +49,10 @@ def price_group_name
     price_group&.name || estimated_price_group_name
   end
 
+  def account_display_description
+    account_description
+  end
```

Comment:
is there a plan to add more complexity here? Otherwise I think we can just use the `account_description`